strict-equal.js

```diff
@@ -0,0 +1,6 @@
+function strictEqual(sampleData) {
```

Comment:
I always like to use brackets. The main reason is readability and I've found bugs in systems in the past due to devs updating code and missing details.  There is no right or wrong on this. There are good arguments on both sides of the debate such as at https://wiki.c2.com/?AlwaysUseBracesOnIfThen